legal-api/tests/unit/core/test_filing_diff.py

```diff
@@ -78,7 +82,35 @@ def test_filing_json_diff():
     assert ld == [{
         'newValue': 'Be it resolved, and now it is.',
         'oldValue': 'Be it resolved, that it is resolved to be resolved.',
-        'path': '/filing/specialResolution/resolution'}]
+        'path': RESOLUTION_PATH}]
+
+
+@pytest.mark.parametrize('test_name, value', [
```

Comment:
could add these to the test_utils section, and increase those tests, rather than introduce general tests here?

Reply:
Ok. Done.